nftables_test.sh

```diff
@@ -0,0 +1,3 @@
+go test ./...
```

Comment:
Can you remove this file from the commit?

Making testing a little easier is a good idea in general, but I’d like to avoid establishing precedent of shell scripts :)

Maybe a “make test” target in a Makefile in a separate PR would be the best way forward?

Reply:
> Can you remove this file from the commit?

@stapelberg , 👍  will do prior to removing WIP.

> Maybe a “make test” target in a Makefile in a separate PR would be the best way forward?

@stapelberg , that would be nice! I agree.